CHANGES.md

```diff
@@ -4,6 +4,8 @@
 
 #### _Black_
 
+- `Black` wraps `**` if in math expression with more than 1 op (#2095)
```

Comment:
I like that more verbose wording too! And since it's not updated yet let's grammar-nazi it up just a wee bit: "unless it's *the* only operation". First I thought it read something like "unless it's *just* operation" which confused me a bit. Obvious on closer inspection but worth a correction in my mind 😄

Reply:
Damn this should have been a reply to @ichard26, see his suggested changes above.